packages/react-utils/package.json

```diff
@@ -50,11 +49,11 @@
   "peerDependencies": {
     "@opensrp/store": "^0.0.10",
     "antd": "^4.6.3",
-    "i18next": "^19.8.4",
     "lodash": "^4.17.21",
     "react": "^16.18.1",
     "react-redux": "^7.2.1",
     "react-router": "^5.2.1",
+    "@opensrp/i18n": "0.0.1",
```

Comment:
wasn't the aim to remove these dependencies?

Reply:
`i18next` and `react-i18next`, they are encapsulated in `@opensrp/i18n`.